packages/file-collections-sa-gridfs/package.json

```diff
@@ -87,6 +86,9 @@
     "@babel/preset-env": "^7.10.4",
     "babel-core": "^7.0.0-bridge.0"
   },
+  "peerDependencies": {
+    "mongodb": "4.4.1"
```

Comment:
I find it best to be loose with peer dependency versions because now that npm fails on mismatched peer deps (I assume pnpm does too), you would need to continually update this range to prevent troubles for consumers of this package. So maybe `"mongodb": ">= 4.4.1 < 5"`